### PR TITLE
[14.0][MIG]sale_customer_rank: Migration to 14.0

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
Standard migration
@ForgeFlow

### CODE REVIEW

File: sale_customer_rank/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+from . import models
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    partners = env["sale.order"].search([]).mapped("partner_id")
+    partners |= partners.mapped("commercial_partner_id")
+    partners._increase_rank("customer_rank")
```
Comment (ChrisOForgeFlow) [inline sale_customer_rank/__init__.py]: Please add specific rank related with number of sales orders related, with parameter n on function _increase_rank

Comment (GuillemCForgeFlow) [reply sale_customer_rank/__init__.py]: Done! :+1:

**Review by ajaniszewska-dev — APPROVED** [review]
LG

**Comment by SerM11** [comment]
We want to test this pr but runbot/runboat is not working.

**Comment by GuillemCForgeFlow** [comment]
Hi @rousseldenis 
Rebased! :+1:

**Comment by rousseldenis** [comment]
@GuillemCForgeFlow Could you check why animal module is making tests failing ?

**Comment by GuillemCForgeFlow** [comment]
Hi @rousseldenis It seems to be related with the id of one menu added in the animal module. If I change the id, it works fine. What should we do here?

**Comment by rousseldenis** [comment]
@GuillemCForgeFlow Could you rebase ?

**Comment by GuillemCForgeFlow** [comment]
> @GuillemCForgeFlow Could you rebase ?

Sure, done!

File: sale_customer_rank/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{
+    "name": "Sale Customer Rank",
+    "summary": "Update Customer Rank when creating a Sale Order",
+    "version": "14.0.1.0.0",
+    "category": "Sales",
+    "website": "https://github.com/OCA/partner-contact",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["sale"],
```
Comment (JuanyDForgeflow) [inline sale_customer_rank/__manifest__.py]: change for sale_management so it is installed

Comment (GuillemCForgeFlow) [reply sale_customer_rank/__manifest__.py]: Sure, done! :+1:

**Comment by rousseldenis** [comment]
@GuillemCForgeFlow Do you plan to add a little test here ?

**Comment by GuillemCForgeFlow** [comment]
> @GuillemCForgeFlow Do you plan to add a little test here ?

Yes, sure I'll add it!

**Comment by GuillemCForgeFlow** [comment]
> @GuillemCForgeFlow Do you plan to add a little test here ?

Done! :+1:

**Comment by GuillemCForgeFlow** [comment]
Hi @rousseldenis , can we get this merged?

File: sale_customer_rank/__init__.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from . import models
+from . import tests
```
Comment (HviorForgeFlow) [inline sale_customer_rank/__init__.py]: Test import on __init__ module file is not used

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: sale_customer_rank/__init__.py (hunk 3 above)
Comment (GuillemCForgeFlow) [reply sale_customer_rank/__init__.py]: It's importing the content from the tests folder. What do you mean exactly?

Comment (HviorForgeFlow) [reply sale_customer_rank/__init__.py]: Check any other Odoo module, you will see tests folder importing does not appear

Comment (GuillemCForgeFlow) [reply sale_customer_rank/__init__.py]: Okey, I see what you mean. Thank you 😄

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**